### ISSUE TITLE
Bad hover hints for parameters in the interface blocks

### ISSUE THREAD

**Issue by automaticp** [nolanderc/glsl_analyzer#8]
glsl_analyzer v1.0.8

The hover hint seems to confuse the parameters between `in` and `out` interface blocks as the same variable if their names match.

The snippet:

```glsl
#version 430 core


in Interface {
    vec2 tex_coords;
} in_[3];

out Interface {
    vec4 frag_pos;
    vec2 tex_coords;
} out_;


void main() {
    out_.tex_coords;
}
```

The hint I get:

![image](https://github.com/nolanderc/glsl_analyzer/assets/83063079/75bef92c-1083-43e6-8a7e-1ac83c35dd74)

**Comment by automaticp** [comment]
Btw, sorry, by matching names I meant the names of the parameters, not the interface blocks themselves. This still happens if the blocks have different names.

**Comment by nolanderc** [comment]
This should be fixed in (v1.0.10)[https://github.com/nolanderc/glsl_analyzer/releases/tag/v1.0.10]
